src/Pact/Types/KeySet.hs

```diff
@@ -192,22 +192,18 @@ instance Arbitrary KeySetName where
 instance Pretty KeySetName where
   pretty ksn = "'" <> pretty (asString ksn)
 
-keysetNameParser
-  :: TokenParsing m
-  => Monad m
-  => m KeySetName
+keysetNameParser :: Parser KeySetName
 keysetNameParser =
   qualifiedKeysetNameParser <|> woNs
   where
-    woNs = (`KeySetName` Nothing) <$> (ident style <* eof)
+    -- legacy support for keyset names must support both
+    -- whitespace spaces as well as names like "SB <2>".
+    woNs = (`KeySetName` Nothing) <$> (takeText <* eof)
```

Comment:
this doesn't need to be followed by EOF.

Reply:
+1